tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Get.feature

```diff
@@ -1,67 +1,75 @@
 ﻿Feature: Get an Order
-    As an Buyer user
-    I want to be able to view a preview of a given order
-    So that I can ensure that the information is complete
-
-Background:
-    Given Addresses exist
+    As an Buyer user
+    I want to be able to view a preview of a given order
+    So that I can ensure that the information is complete
+
+Background:
+    Given Addresses exist
         | Line1 | Line2      | Line3      | Line4          | Line5           | Town      | County  | Postcode | Country        |
         | 4     | Upper Flat | Rocks Lane | Little Village | Bigger Village  | Some Town | W Yorks | LS1 3AP  | United Kingdom |
-        | 8     | Lower Flat | Water Lane | Big Village    | Smaller Village | The Town  | E Yorks | LS2 6ZP  | United Kingdom |
```

Comment:
I think the line endings might have messed up on this file, makes it kinda awkward to figure out what's actually changed